src/client/components/layout/InstructionalTextBox.tsx

```diff
@@ -0,0 +1,13 @@
+import { fromTheme } from 'mark-one';
+import styled from 'styled-components';
+
+const StyledInstructionalTextBox = styled.div`
+  border: ${fromTheme('border', 'light')};
+  margin-top: ${fromTheme('ws', 'xsmall')};
+  margin-bottom: ${fromTheme('ws', 'xsmall')};
+  padding: ${fromTheme('ws', 'xsmall')};
+  font-family: ${fromTheme('font', 'note', 'family')};
+  font-size: ${fromTheme('font', 'note', 'size')};
```

Comment:
Can we add a little spacing under the child `p` elements, e.g.

```
  & p {
    margin-bottom: 0.5em;
  }
`;
```

Reply:
Sure thing, I updated the styling in [22425eb](https://github.com/seas-computing/course-planner/pull/529/commits/22425eb84bfa9d1871a6ff80d566cc5d4cda1c08).